navit/vehicleprofile.c

```diff
@@ -211,8 +213,8 @@ vehicleprofile_update(struct vehicleprofile *this_)
 			vehicleprofile_apply_attrs(this_, profile_option.u.navit_object, 1);
 	}
 	vehicleprofile_attr_iter_destroy(iter);
-	dbg(lvl_debug,"result l %d w %d h %d wg %d awg %d pen %d\n",this_->length,this_->width,this_->height,this_->weight,this_->axle_weight,this_->through_traffic_penalty);
```

Comment:
Hmm, do we really want those messages on error level? Suppose to change them to debug before merge.